qcodes/dataset/data_set.py

```diff
@@ -949,22 +950,17 @@ def _parameter_data_identical(param_dict_a: Dict[str, numpy.ndarray],
 
     def _same_setpoints(self, datadict: ParameterData) -> bool:
 
-        def _get_setpoints(dd: ParameterData) -> Iterator[Dict[str, numpy.ndarray]]:
+        def _sp_iterator(dd: ParameterData) -> Iterator[Dict[str, numpy.ndarray]]:
 
             for dep_name, param_dict in dd.items():
                 out = {
                     name: vals for name, vals in param_dict.items() if name != dep_name
                 }
                 yield out
 
-        iterator = _get_setpoints(datadict)
```

Comment:
I actually meant having a variable called iterator is not very descriptive

Reply:
ahh thanks, agreed. I'll get that name change in.